tests/func-tests/node_placement_test.go

```diff
@@ -63,6 +63,7 @@ var _ = Describe("[rfe_id:4356][crit:medium][vendor:cnv-qe@redhat.com][level:sys
 				//"kube-multus":     false,
 				"ovs-cni-marker": false,
 				"virt-handler":   false,
+				"secondary-dns":  false,
```

Comment:
need to do also like https://github.com/kubevirt/hyperconverged-cluster-operator/pull/2195/commits/0a925dd25b95fbd6ad9c3c397f1c392e4e62d6de#diff-ed5f0c7b958324db1687cbf20e18758696850ec0fc30c638cf913f523a3452edR71

so in case we dont deploy it, the test will pass